backend/controllers/feedController.js

```diff
@@ -193,6 +194,24 @@ const getUserInfo = async (req, res, next) => {
 };
 
 const getAllUsersInfo = async (req, res, next) => {
+  const { search, orderBy } = req.query;
+  let searchQuery;
+  let orderByQuery;
+  if (search.length > 0) {
+    searchQuery = {
+      [Op.or]: {
+        first_name: { [Op.startsWith]: search },
+        last_name: { [Op.startsWith]: search },
+      },
+    };
+  } else {
+    searchQuery = { first_name: { [Op.ne]: null } };
+  }
+  if (orderBy.length > 0) {
+    orderByQuery = [["first_name", orderBy]];
+  } else {
+    orderByQuery = [["id", "ASC"]];
+  }
```

Comment:
- there is no need for this conditions. Please put the arguments directly in _line:235_. To shorten this one you can use ternary operator `??`. 
ex. `order: getThisIfNotNull ?? getThisIfNull`
- and why 1st argument is not same.